### PR TITLE
CTE tutorial improvements

### PR DESCRIPTION
**Pull request by lfunderburk**
## Describe your changes

Introduced catchy intro, along with a summary of what JupySQL does in the example.

## Issue ticket number and link
Closes https://github.com/ploomber/jupysql/issues/280

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I have added thorough [tests](https://github.com/ploomber/contributing/blob/main/CONTRIBUTING.md#testing) (when necessary).
- [x] I have added the right documentation in the [docstring](https://github.com/ploomber/contributing/blob/main/CONTRIBUTING.md#documenting-changes-and-new-features) and [changelog](https://github.com/ploomber/contributing/blob/main/CONTRIBUTING.md#changelog) (when needed)

Preview https://jupysql--300.org.readthedocs.build/en/300/compose.html

### CODE REVIEW

**Comment by lfunderburk** [comment]
@edublancas ready for review

File: doc/compose.md [hunk 1]
```diff
@@ -33,10 +33,16 @@ This is a beta feature, please [join our community](https://ploomber.io/communit
 let us know how we can improve it!
 ```
 
-JupySQL allows you to break queries into multiple cells, simplifying the process of
-building large queries.
+Tired of wrangling complex, large-scale SQL queries? JupySQL is here to revolutionize your experience with an intuitive, user-friendly package specifically designed to help you manage and organize large SQL queries in Jupyter Notebooks.
```
Comment (edublancas) [inline doc/compose.md:36]: As a general rule, we keep the tone in our docs non sales-y, and more direct.

File: doc/compose.md [hunk 2]
```diff
@@ -33,10 +33,16 @@ This is a beta feature, please [join our community](https://ploomber.io/communit
...
 
-As an example, we are using a sales database from a record store. We'll find the
+- **Simplify your workflow:** JupySQL allows you to break down large SQL queries into manageable chunks and organize them within your Jupyter Notebook, enhancing readability and maintainability.
```
Comment (edublancas) [inline doc/compose.md:38]: I think this list is good, but let's make it shorter

File: doc/compose.md [hunk 3]
```diff
@@ -33,10 +33,16 @@ This is a beta feature, please [join our community](https://ploomber.io/communit
...
+- **Modularize your queries:** With JupySQL, you can create reusable query modules and leverage them across your project, promoting code reusability and reducing redundancy.
+- **Seamless integration:** JupySQL flawlessly combines the power of SQL with the flexibility of Jupyter Notebooks, offering a one-stop solution for all your data analysis needs.
+- **Cross-platform compatibility:** JupySQL supports popular databases like PostgreSQL, MySQL, SQLite, and more, ensuring you can work with any data source.
+
+## Example: record store data
```
Comment (edublancas) [inline doc/compose.md:43]: are there any other improvements we can do to the rest of the file? I think the general style and flow could be improved. For a better dev experience, ensure you're able to run this from Jupyter: https://github.com/ploomber/contributing/blob/main/documentation/notebooks.md

**Review by edublancas — CHANGES_REQUESTED** [review]

File: doc/compose.md (hunk 1 above)
Comment (lfunderburk) [reply doc/compose.md]: Noted. Resolve in https://github.com/ploomber/jupysql/pull/300/commits/50219a0c10c5aa8bc64bbbd849e4e41f945ade1d

File: doc/compose.md (hunk 2 above)
Comment (lfunderburk) [reply doc/compose.md]: Resolved in https://github.com/ploomber/jupysql/pull/300/commits/50219a0c10c5aa8bc64bbbd849e4e41f945ade1d

**Comment by lfunderburk** [comment]
Hello @edublancas  and @idomic 

Added new suggestions to improve flow. Also noticed that when the docs are built, the last code block calling $final would return an error message. 

```
%%sql
$final
*  sqlite:///my.db
(sqlite3.OperationalError) unrecognized token: "{"
[SQL: {final}]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
```

I executed locally on a notebook and obtained this warning:

```
FutureWarning: Variable substitution with $var and {var} has been deprecated and will be removed in a future version. Use {{var}} instead. To remove this, see: https://jupysql.ploomber.io/en/latest/howto.html#ignore-deprecation-warnings
  warnings.warn(msg, FutureWarning)
```

Changed the block to 

```
%%sql
{{final}}
```

File: doc/compose.md (hunk 3 above)
Comment (lfunderburk) [reply doc/compose.md:42]: Attempted new solution here

**Review by edublancas — APPROVED** [review]

**Comment by edublancas** [comment]
great work!

> Added new suggestions to improve flow. Also noticed that when the docs are built, the last code block calling $final would return an error message.

yes, this makes sense since we deprecated the old `$var` behavior!

**Pull request merged**